src/rai_bench/rai_bench/tool_calling_agent/tasks/spatial.py

```
@@ -14,18 +14,36 @@
+SPATIAL_REASONING_SYSTEM_PROMPT_0_SHOT = """You are a helpful and knowledgeable AI assistant that specializes in interpreting and analyzing visual content. Your task is to answer questions based on the images provided to you. Please response with the use of the provided tools."""
+
+SPATIAL_REASONING_SYSTEM_PROMPT_2_SHOT = (
+    SPATIAL_REASONING_SYSTEM_PROMPT_0_SHOT
+    + """
+
+Example of tool calls:
+- return_bool_response, args: {'response': True}
+- return_bool_response, args: {'response': False}"""
+)
+
+# NOTE (jmatejcz) In this case we are using only one tool so there is no difference bettween 2 and 5 shot
```

Flake8 (7.2.0)

[error] line too long (105 > 79 characters)

(E501)